resources/indexer.py

```diff
@@ -405,12 +565,29 @@ def gather_file_info(dirs, site):
             yield from file_meta
 
 
+async def request_post_patch(fc_rc, metadata):
+    """POST metadata, and PATCH if file is already in the file catalog."""
+    try:
+        logging.debug('POSTing...')
+        _ = await fc_rc.request("POST", '/api/files', metadata)
+        logging.debug('POSTed.')
+    except requests.exceptions.HTTPError as e:
+        if e.response.status_code == 409:
+            patch_path = e.response.json()['file']  # /api/files/{uuid}
+            logging.debug('PATCHing...')
```

Comment:
I would be specific in the log: PATCH {patch_path}

Reply:
I've removed this. It's already being logged by urllib3.connectionpool, so no need to double log.